packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -459,7 +459,7 @@ abstract class FlutterCommand extends Command<void> {
         'the information needed to symbolize Dart stack traces. For an app built '
         "with this flag, the 'flutter symbolize' command with the right program "
         'symbol file is required to obtain a human readable stack trace.',
-      valueHelp: '/project-name/v1.2.3/',
+      valueHelp: '../project-name/v1.2.3/',
```

Comment:
just remove `../project-name ` entirely, here and below

Reply:
ok! , i will do it